src/cbc_sdk/workload/vulnerability_assessment.py

```diff
@@ -24,125 +24,33 @@
-
-        Returns:
-            VulnerabilityQuery: The query object for this vulnerability type.
-        """
-        return VulnerabilityQuery(cls, cb)
-
-
-class VulnerabilitySummary(NewBaseModel):
-    """Represents a vulnerability summary."""
-    urlobject = "/vulnerability/assessment/api/v1/orgs/{0}"
-    swagger_meta_file = "workload/models/organizationVulnerabilitySummary.yaml"
-    url_additional = "/summary"
-
-    VALID_SEVERITY = ["CRITICAL", "IMPORTANT", "MODERATE", "LOW"]
-
-    def __init__(self, cb, model_unique_id=None, initial_data=None):
-        """
-        Initialize VulnerabilitySummary object
-
-        Args:
-            cb (BaseAPI): Reference to API object used to communicate with the server.
-            model_unique_id (int): Not used by this class
-            initial_data (dict): dictionary of the data
-        """
-        super(VulnerabilitySummary, self).__init__(cb, model_unique_id, initial_data)
-        self._full_init = True
-
-    @classmethod
-    def _query_implementation(cls, cb, **kwargs):
-        """
-        Returns the appropriate query object for this object type.
-
-        Args:
-            cb (BaseAPI): Reference to API object used to communicate with
-            the server.
-            **kwargs (dict): Not used, retained for compatibility.
-
-        Returns:
-            VulnerabilityQuery: The query object for this vulnerability type.
-        """
-        return VulnerabilityQuery(cls, cb)
-
-    @classmethod
-    def get_org_vulnerability_summary(cls, cb, severity=None, vcenter_id=None):
-        """
-        Returns vulnerability summary at the organization level
-
-        Args:
-            cb (BaseAPI): Reference to API object used to communicate with the server.
-            severity (str): (optional) filters the vulnerability sumary per severity
-            (CRITICAL, IMPORTANT, MODERATE, LOW)
-            vcenter_id (str): (optional) filters the vulenerability per vcenter id
-        Returns:
-            dict: monitored_assets and severity summary
-        """
-        query_params = {}
-        if vcenter_id:
-            url = "/vcenters/{}/vulnerabilities/summary".format(vcenter_id)
-        else:
-            url = "/vulnerabilities/summary"
-
-        if severity and severity not in VulnerabilitySummary.VALID_SEVERITY:
-            raise ApiError("Invalid severity")
-        elif severity:
-            query_params["severity"] = severity
-
-        req_url = OrganizationalVulnerability.urlobject.format(
-            cb.credentials.org_key) + url
-        return cb.get_object(req_url, query_params)
-
-
 class Vulnerability(NewBaseModel):
     """Represents a vulnerability"""
-    urlobject = '/vulnerability/assessment/api/v1'
-    urlobject_single = '/vulnerability/assessment/api/v1/vulnerabilities/{}'
+    urlobject = '/vulnerability/assessment/api/v1/orgs/{}'
     swagger_meta_file = "workload/models/vulnerability.yaml"
     url_additional = ""
```

Comment:
I'm sorry I did not notice this earlier, but I think url_additional is not needed any more.

Reply:
Done!